frontend/src/components/SupportIconsBar.tsx

```diff
@@ -13,17 +13,16 @@
     limitations under the License.
 */
 
-import { Link } from "@mui/material";
-import { Stack } from "@mui/material";
+import { Link, Stack, Tooltip} from "@mui/material";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faSlack, faGithub } from "@fortawesome/free-brands-svg-icons";
 import { faBook } from "@fortawesome/free-solid-svg-icons";
-import { Tooltip } from "@mui/material";
+import { SxProps } from "@mui/system";
 import { SizeProp } from "@fortawesome/fontawesome-svg-core";
 
 interface SupportIconsBarProps {
-  sx: { [key: string]: string | number};
-  iconsSize?: SizeProp;
+  sx?: SxProps;
+	iconsSize?: SizeProp;
```

Comment:
For some reason this indentation is not reflecting on my local changes. I tried making changes to my vs code formatter settings.

Reply:
No problem we can remove the indentation in a refactor at some point